rdock-utils/rdock_utils/rbhtfinder/rbhtfinder.py

```diff
@@ -0,0 +1,357 @@
+import itertools
+import logging
+import multiprocessing
+import os
+from collections import Counter, defaultdict
+from functools import partial
+
+import numpy as np
+import pandas as pd
```

Comment:
If pandas is not installed, this line will raise an ImportError exception.  
We need to handle this and set a variable (something like `PANDAS_IS_AVAILABLE`) to `True` if the import is successful, or `False` otherwise, and use it below.  

Anyways, I'd rather not add pandas as a dependency. We will explore the possible ways to do it together.